eng/common/scripts/tag-reviewer.ps1

```diff
@@ -0,0 +1,61 @@
+param(
+    [Parameter(Mandatory = $true)]
+    $RepoOwner,
+
+    [Parameter(Mandatory = $true)]
+    $RepoName,
+
+    [Parameter(Mandatory = $true)]
+    $GitHubUser,
```

Comment:
We should consider accepting a list of reviewers.

Reply:
Completed. Ended up being a bit more finicky than I expected :P